h/static/scripts/polyfills.js

```diff
@@ -40,3 +40,23 @@ require('keyboardevent-key-polyfill').polyfill();
 // Fetch API
 // https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API
 require('whatwg-fetch');
+
+// Quick polyfill to normalize the naming of matches function
+// from https://developer.mozilla.org/en/docs/Web/API/Element/matches
+if (!Element.prototype.matches) {
```

Comment:
If we use the Element.closest API as suggested in https://github.com/hypothesis/h/pull/4174/files#r91678734 , we can remove this polyfill and replace it with `element-closest`.